src/main/java/sg/ncl/MainController.java

```diff
@@ -60,8 +60,12 @@
 
     private String AUTHORIZATION_HEADER = "Basic dXNlcjpwYXNzd29yZA==";
 
+    private final static String CONTACT_EMAIL = "support@ncl.sg";
+
     // error messages
-    private final String ERR_SERVER_OVERLOAD = "There is a problem with your request. Please contact support@ncl.sg";
+    private final static String ERR_SERVER_OVERLOAD = "There is a problem with your request. Please contact " + CONTACT_EMAIL;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)